zebrad/src/commands/generate.rs

```diff
@@ -31,8 +31,16 @@ impl Runnable for GenerateCmd {
 # can be found in Rustdoc here:
 # https://doc.zebra.zfnd.org/zebrad/config/struct.ZebradConfig.html
 
+# Usage:
+# This file must be located in the same directory zebrad binary is
+# being called, it is loaded by the -c flag.
+# For example if you generate with:
+# ./zebrad generate -o zebrad.toml
+# Do edits to the file as needed then execute the following to connect
+# using the new configuration, default values will be overwritten:
+# ./zebrad -c zebrad.toml connect
```

Comment:
I think we implicitly load the `zebrad.toml` in the current working directory if one exists, and that we should mention this in the usage.

Reply:
Yep, and:
* probably better to write `zebrad` instead of `./zebrad`;
* the other part of the source code where the default config path is specified should have a comment reminding people to update this help text if they change it.

Reply:
and in case it helps the code that defines the path is in `command.rs`

Reply:
Thank you for the comments guys, i added all of them and force pushed.